src/main/resources/BridgeServer2.conf

```diff
@@ -147,7 +147,10 @@ uat.udd.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-
 prod.udd.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-WorkerPlatform-Request-prod
 
 #Bridge Participant Roster Download Worker SQS queues
-workerPlatform.request.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/420786776710/Bridge-WorkerPlatform-Request-local
+local.workerPlatform.request.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/420786776710/Bridge-WorkerPlatform-Request-local
+dev.workerPlatform.request.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/420786776710/Bridge-WorkerPlatform-Request-develop
+uat.workerPlatform.request.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/420786776710/Bridge-WorkerPlatform-Request-uat
+prod.workerPlatform.request.sqs.queue.url = https://sqs.us-east-1.amazonaws.com/420786776710/Bridge-WorkerPlatform-Request-prod
```

Comment:
uat should be https://sqs.us-east-1.amazonaws.com/420786776710/Bridge-WorkerPlatform-Request-staging

prod should be https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-WorkerPlatform-Request-prod